### ISSUE TITLE
Alternativ med ren REST tilnærming 

### ISSUE THREAD

**Issue by thomiz** [HL7Norway/innrapportering-helseregister#1]
Som foreslått for eDår fase2 API
Kan beskrives som mulig tilnærming istedenfor eller i tillegg til Composition Bundle?

![image](https://user-images.githubusercontent.com/6229665/97442801-87a52500-192a-11eb-8c25-85e9a0db47d8.png)

**Comment by thomiz** [comment]
Composition vs REST tilnærming
![image](https://user-images.githubusercontent.com/6229665/97443469-55e08e00-192b-11eb-99c9-f9b4e08b748a.png)

**Comment by rockphotog** [comment]
Ja, jeg tror vi må beskrive begge "patterns". For noen typer "må" det kanskje være et dokument, men mange dokumenter kan med fordel være ren REST for å tenke "API first" for hhv tett integrasjon EPJ, portal, app etc. -- og "lag dokument" kan også være en tjeneste/operation server side (så kan man laste det ned for å ha en kopi/snapshot). 

I målarkitektur for datadeling/helseregistre må det utredes hva som er den foretrukne metoden. 

Vi bør også visualisere et typisk REST mønster der alltid prøver GET Patient før man evt PUT-er, etc.
